Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
```

Comment:
I quite like `ECS_CNI_REPOSITORY_REVISION` but I wanted the target names to be short too. 
I'm sorry if there has been a misunderstanding here. 

Ex:
`make get-amazon-ecs-cni-sources ECS_CNI_REPOSITORY_REVISION=dev` is too long imho

I'm still in favor of having the below mentioned targets:
* cni-sources
* cni-plugins

This also helps us in not binding everything down to the amazon-ecs-cni-plugins.